examples/complex_type.cpp

```diff
@@ -83,6 +83,14 @@ int main()
     std::cout << "Enumeration::C: " << value << std::endl;
     std::cout << "Enumeration2::C: " << value2 << std::endl;
 
+    std::cout << "-----------------------" << std::endl;
+    submod_b.enum_32(std::move(my_enum));
+     root.create_constant("MyConstEnum", enum_data); // TODO: cast enumerations
```

Comment:
What do you mean with _cast enumerations_?

Reply:
See the suggestion in the first commit. It was an old comment!